proteuslib/unit_models/tests/test_reverse_osmosis_0D.py

```diff
@@ -649,23 +655,37 @@ def test_Pdrop_calculation(self):
         assert results.solver.status == SolverStatus.ok
 
         # test solution
-        assert (pytest.approx(-9.173e5, rel=1e-3) == value(m.fs.unit.deltaP[0]))
-        assert (pytest.approx(1.904e-3, rel=1e-3) ==
+        assert (pytest.approx(151.622, rel=1e-3) == value(m.fs.unit.N_Re_io[0, 'in']))
+        assert (pytest.approx(0.2089, rel=1e-3) == value(m.fs.unit.velocity_io[0, 'in']))
+        assert (pytest.approx(63.001, rel=1e-3) == value(m.fs.unit.N_Re_io[0, 'out']))
+        assert (pytest.approx(0.0914, rel=1e-3) == value(m.fs.unit.velocity_io[0, 'out']))
+        assert (pytest.approx(151.622, rel=1e-3) == value(m.fs.unit.velocity_io[0, 'in']
+                * m.fs.unit.feed_side.properties_in[0].dens_mass_phase['Liq']
+                * m.fs.unit.dh
+                / m.fs.unit.feed_side.properties_in[0].visc_d_phase['Liq']))
+        assert (pytest.approx(63.001, rel=1e-3) == value(m.fs.unit.velocity_io[0, 'out']
+                * m.fs.unit.feed_side.properties_out[0].dens_mass_phase['Liq']
+                * m.fs.unit.dh
+                / m.fs.unit.feed_side.properties_out[0].visc_d_phase['Liq']))
```

Comment:
This test looks like it was part of your debugging for Reynolds number.

Reply:
Yeah I understand, but I take that more as a debugging activity. We rarely have two tests for a variable and another way to calculate it. 

For example you could check that the average flux equals the permeate flow over the membrane area or that the velocity equals the total mass flowrate divided by density and cross sectional area (its currently being calculated by total flowrate divided by cross sectional area). The combinations of calculating variables in equivalent ways would be really high and its arbitrary that we are just doing this for the Reynolds number calculation, so I'd leave this just as a debugging/model development exercise rather than a test.

Reply:
Ok, makes sense.